tests/test_view_decorators.py

```diff
@@ -78,6 +78,7 @@ def test_fresh_jwt_required(app):
     with app.test_request_context():
         access_token = create_access_token('username')
         fresh_access_token = create_access_token('username', fresh=True)
+        fresh_timed_access_token = create_access_token('username', fresh=timedelta(minutes=-1))
```

Comment:
Could we add one more unit test somewhere? You have the not fresh test case here, I would like to have a test added that sets the time delta in the future and successfully accesses a fresh jwt required endpoint while using this new feature. Thanks!